ee/packages/federation-matrix/tests/scripts/run-integration-tests.sh

```
@@ -141,20 +164,12 @@ cleanup() {
         if [ "$INCLUDE_ELEMENT" = true ]; then
             log_info "  - Element: https://element"
         fi
-        if [ "$INCLUDE_ELEMENT" = true ]; then
-            log_info "To stop containers manually, run: docker compose -f $DOCKER_COMPOSE_FILE --profile element down -v"
-        else
-            log_info "To stop containers manually, run: docker compose -f $DOCKER_COMPOSE_FILE --profile test down -v"
-        fi
+        log_info "To stop containers manually, run: docker compose -f "$DOCKER_COMPOSE_FILE" --profile "$COMPOSE_PROFILE" down -v"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)